### PR TITLE
Text Filters

### PR DESCRIPTION
**Pull request by gwincr11**
Motivation:
  - https://github.com/smpallen99/ex_admin/issues/49

Changes:
  - Use the new string filters available in ex_queb
    https://github.com/E-MetroTel/ex_queb/pull/5
  - Add the new filter options to the select options
  - Setup tests and styles.

### CODE REVIEW

**Comment by gwincr11** [comment]
So, I am using webpack in my local setup. For some reason the .css.css activeadmin file does not work with this setup. So I couldn't properly test the activeadmin theme's styling, please take a look.

File: lib/ex_admin/themes/admin_lte2/filter.ex [hunk 1]
```diff
@@ -7,6 +7,7 @@ defmodule ExAdmin.Theme.AdminLte2.Filter do
   import ExAdmin.Gettext
   import ExAdmin.Filter
   use Xain
+  require IEx
```
Comment (smpallen99) [inline lib/ex_admin/themes/admin_lte2/filter.ex]: This should be removed

**Comment by smpallen99** [comment]
@gwincr11 Default does not work. Default select box says "Contains", but the filter applied is _eq. 

I'll also release a new ex_queb to hex so this will pickup the correct version of ex_queb.

**Comment by gwincr11** [comment]
@smpallen99 should the default be equals or contains?

File: lib/ex_admin/themes/admin_lte2/filter.ex (hunk 1 above)
Comment (gwincr11) [inline lib/ex_admin/themes/admin_lte2/filter.ex]: Opps, thanks

**Comment by gwincr11** [comment]
@smpallen99 ok, I have it working with the default now.

**Pull request merged**